daemon/appinfo.c

```diff
@@ -851,8 +885,15 @@ appinfo_parse_desktop(appinfo_t *self)
         group = SAILJAIL_SECTION_SECONDARY;
     /* else: legacy app => use default profile */
 
+    /* Sandboxing=Disabled means that the app opts out of sandboxing and
```

Comment:
`None` or `Disabled`?

Reply:
I mean the mode is called "None", while the option for it has the value "Disabled". Should we have it more consistent?

Reply:
(As well as apps are called "legacy", while the mode is called "compatibility".)

Reply:
And "Enabled" vs "Normal".

Reply:
Well, yeah, I didn't want to couple this Mode value and the opt out option but maybe I should have. It crossed my mind that the key could be also Sandboxing in GetAppInfo output and then the values would be Enabled, Compatibility, Disabled or something to that direction. It would make it possible to use Compatibility mode even with a sandboxing aware app unless it's prevented (and of course desktop file checks in validator will prevent any unwanted options like opt out).

And I could name the mode differently if needed too.